hack/test-go.sh

```diff
@@ -324,7 +324,12 @@ for (( i=0, j=0; ; )); do
   etcdPrefix=${etcdPrefixes[j]}
   echo "Running tests for APIVersion: $apiVersion with etcdPrefix: $etcdPrefix"
   # KUBE_TEST_API sets the version of each group to be tested.
-  KUBE_TEST_API="${apiVersion}" ETCD_PREFIX=${etcdPrefix} runTests "$@"
+  time {
+   kube::log::status "Starting tests at '$(date)'"
```

Comment:
`kube::log::status` includes a timestamp, so $(date) is a little redundant, though not that big a deal.